specification/components/schemas/headers/request/ContentTypeFhirJson.yaml

```diff
@@ -0,0 +1,9 @@
+in: header
+name: Content-Type
+description: |
+  Indicates the media type of the associated resource. 
+schema:
+  type: string
+  enum:
```

Comment:
Not sure if this should be an enum or not.

Reply:
I think this makes sense as an enum here really as for most endpoints you'll need to provide the content type "application/fhir+json", it's just the File Upload endpoint that utilises a different content type.